cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinder.java

```diff
@@ -286,6 +297,16 @@ private void detectMetricAnomaliesFromPeers(Map<BrokerEntity, Double> currentVal
     }
   }
 
+  private Set<BrokerEntity> detectMetricAnomaliesFromThreshold(Map<BrokerEntity, Double> currentValue, double threshold) {
+    Set<BrokerEntity> detectedMetricAnomalies = new HashSet<>();
+    for (Map.Entry<BrokerEntity, Double> entry : currentValue.entrySet()) {
```

Comment:
The loop uses only the key, but iterates over entities. Can we either iterate over `keySet()` or update the logic to use `getValue()`, as well?